src/indicators/spinner.js

```diff
@@ -28,10 +28,12 @@ import classnames from 'classnames'
  */
 
 const propTypes = {
-  className: ''
+  className: PropTypes.string,
 }
 
-const defaultProps = {}
+const defaultProps = {
+  className: 'spinner'
```

Comment:
If `className` isn't provided, your div will look like this:
```jsx
<div class="spinner spinner">
```
An empty string would be a more appropriate default (ignored by `classnames` util)